frontend/src/hooks/api/workspace/queries.tsx

```diff
@@ -450,3 +454,89 @@ export const useGetWorkspaceIdentityMemberships = (workspaceId: string) => {
     enabled: true
   });
 };
+
+export const useAddGroupToWorkspace = () => {
```

Comment:
This function is in the queries.tsx file, but it's a mutation not a query. This should be moved to a mutation.tsx file.